src/onefetch/cli.rs

```diff
@@ -115,6 +116,16 @@ impl Cli {
                 .possible_values(&possible_backends)
                 .help("Which image BACKEND to use."),
         )
+        .arg(
+            Arg::with_name("image-colors")
+                .long("image-colors")
+                .value_name("NUM")
+                .takes_value(true)
+                .max_values(1)
+                .possible_values(&["16", "32", "64", "128", "256"])
+                .default_value("16")
+                .help("NUM of colors [16, 32, 64, 128, 256] used in image backend."),
```

Comment:
```suggestion
                .help("NUM of colors [16, 32, 64, 128, 256] to use in image backend."),
```
Minor nitpick: this makes the grammar among the option arg help more consistent.